pkg/util/dockerutils.go

```diff
@@ -219,3 +219,30 @@ func GetContainerHealth(container docker.APIContainers) string {
 	}
 	return match
 }
+
+// GetAppContainers retrieves docker containers for a given sitename.
+func GetAppContainers(sitename string) []docker.APIContainers {
+	label := map[string]string{"com.ddev.site-name": sitename}
+	sites, err := FindContainersByLabels(label)
+	if err != nil {
+		log.Fatal("failed to retrieve containers for ", sitename, err)
+	}
+	return sites
+}
+
+// GetContainerEnv returns the value of a given environment variable from a given container.
+func GetContainerEnv(key string, container docker.APIContainers) string {
+	client := GetDockerClient()
+	inspect, err := client.InspectContainer(container.ID)
+	if err != nil {
+		log.Debug("failed to retreive container ", ContainerName(container))
+	}
+	envVars := inspect.Config.Env
+
+	for _, env := range envVars {
+		if strings.HasPrefix(env, key) {
+			return strings.TrimPrefix(env, key+"=")
+		}
+	}
+	return ""
```

Comment:
I think there's a good case for not found being an error, but you have more context than I do on this one